roles/auth_proxy/files/contivRule.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=run contiv rule service 
+
```

Comment:
Do we want a `After=network.target` here?  I'm not sure at what point in systemd's startup we can guarantee the netfilter drivers are loaded

Reply:
and also before docker or whatever service that requires the rules in place

Reply:
If you have this service...just wonder why we have all the rules in the roles?

Reply:
indeed

Reply:
@dseevr I believe, netfilter is part of kernel and should always up with kernel itself before any services anyway  it would be safe to include "After=network.target" .
@tiewei , we don't need to taken care for docker rules since it would be taken care by docker-swarm/docker engine 
@kahou82 Initially i was thinking do same but couldn't quick logic to place it . therefore use existing role infra to put my changes.  Would need some help if want to go   above suggested solution